subworkflows/nf-core/utils_nfcore_pipeline/tests/main.function.nf.test

```diff
@@ -41,58 +41,6 @@ nextflow_function {
         }
     }
 
-    test("Test Function workflowCitation") {
```

Comment:
Removed, why?

Reply:
https://nf-co.re/blog/2024/maintainers-minutes-2024-11-29#removal-of-workflowcitation